### PR TITLE
REV-153-create-listings-implement-special-tags

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-153

### Description

- add tags input to create listing page

### Dependencies in Other PRs,Repo

- backend PR: https://github.com/abukhadraop-or/reverifi-api/pull/40

Screenshot from 2022-02-27 17-51-12

### CODE REVIEW

File: src/components/create-listing/CreateListingForm/index.jsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ function CreateListingForm({ date }) {
     setValue,
     watch,
     reset,
+    control,
```
Comment (mohdabed123) [inline src/components/create-listing/CreateListingForm/index.jsx:43]: update docs for date

File: src/components/home/FeaturedListingCard/index.jsx [hunk 2]
```diff
@@ -70,6 +71,7 @@ function Card({ data }) {
           <TagContainer>
             {getDatesDifference(createdAt, 7) && <Tag isNew>New</Tag>}
             {listingType?.type && <Tag>{listingType?.type}</Tag>}
+            {tags?.length > 0 && tags.map((tag) => <Tag key={tag}>{tag}</Tag>)}
```
Comment (mohdabed123) [inline src/components/home/FeaturedListingCard/index.jsx:74]:
```suggestion
            {tags && tags.map((tag) => <Tag key={tag}>{tag}</Tag>)}
```

File: src/components/listing-search-page/Card/index.jsx [hunk 3]
```diff
@@ -55,7 +57,12 @@ function Card({ data }) {
   return (
     <CardContainer onClick={handleClick}>
       <ImageContainer>
-        <Badge>Sale</Badge>
+        <BadgesContainer>
+          <Badge>Sale</Badge>
+          {tags?.length > 0 &&
```
Comment (mohdabed123) [inline src/components/listing-search-page/Card/index.jsx:62]:
```suggestion
          {tags &&
```

**Review by mohdabed123 — CHANGES_REQUESTED** [review]

File: src/components/home/FeaturedListingCard/index.jsx (hunk 2 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/home/FeaturedListingCard/index.jsx:74]: tags alone won't work cuz it's array

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**